library/patient.inc.php

```diff
@@ -1350,8 +1325,8 @@ function newInsuranceData(
         $effective_date_end = null;
     }
 
-    $idres = sqlStatement("SELECT * FROM insurance_data WHERE " .
-    "pid = ? AND type = ? ORDER BY date DESC", array($pid,$type));
+    /* $idres = sqlStatement("SELECT * FROM insurance_data WHERE " .
```

Comment:
better to remove than comment out (makes it harder to follow)

Reply:
Fixed